groups/live-windows-virtual-machines/profiles/heritage-live-eu-west-2/vars

```diff
@@ -95,4 +95,19 @@ chips_estor_cw_logs = {
    "system" = {
     log_group_retention = 365
   }
+}
+
+# Business Objects Server 1
+bus_obj_1_application       = "Business Objects Server 1"
+bus_obj_1_ec2_name          = "busobj1-app"
+bus_obj_1_ec2_instance_size = "m5.2xlarge"
+bus_obj_1_ami               = "ami-082ace3b4ab9d3345"
```

Comment:
static ami, should this be from data sources with filters?

Reply:
Not use base AMI, using Windows Server 2019 AMI.